pkg/template/jsonpath.go

```diff
@@ -89,7 +89,16 @@ func textValue(v reflect.Value) ([]byte, error) {
 
 	switch t.Kind() {
 	// evalToText() returns <map> ....; return JSON string instead.
-	case reflect.Map, reflect.Slice:
+	case reflect.String:
```

Comment:
I wonder if we can always use `json.Marshal` and get rid of the `evalToText` altogether....we'd still need a special case for strings to remove the quotes but for every other type I think using `json.Marshal` would give us what we want! 
(Just a thought.....if we decide to do this, we should do it in a followup)

Reply:
ah interesting idea! ill see what happens to our test cases at least :D